hw/ip/uart/doc/checklist.md

```diff
@@ -104,16 +104,16 @@ Security      | [SEC_CM_COUNCIL_REVIEWED][]  | N/A         | This block only con
 Documentation | [NEW_FEATURES_D3][]     | N/A         |
 RTL           | [TODO_COMPLETE][]       | Done        |
 Code Quality  | [LINT_COMPLETE][]       | Done        |
-Code Quality  | [CDC_COMPLETE][]        | N/A         |
-Code Quality  | [RDC_COMPLETE][]        | Not Started |
-Review        | [REVIEW_RTL][]          | Done        | by @msfschaffner
-Review        | [REVIEW_DELETED_FF][]   | N/A         | Not reported by FPGA (@eunchan double-check)
+Code Quality  | [CDC_COMPLETE][]        | N/A         | No errors on top-level
```

Comment:
maybe add the same explanation for CDC/RDC/FF as suggested here:
https://github.com/lowRISC/opentitan/pull/12892#discussion_r904019436